### PR TITLE
Fast Pair headers: new UUID header

### PR DESCRIPTION
**Pull request by kapi-no**

### CODE REVIEW

**Comment by kapi-no** [comment]
A documentation update is planned within the scope of this PR after the technical review (changelog + plugging the UUID API documentation into RST Fast Pair file).

**Review by alwa-nordic — APPROVED** [review]

**Review by alstrzebonski — APPROVED** [review]

**Comment by kapi-no** [comment]
> Please edit codeowners file also to fix compliance fail.

done

**Comment by kapi-no** [comment]
> A documentation update is planned within the scope of this PR after the technical review (changelog + plugging the UUID API documentation into RST Fast Pair file).

Added changelog entries

**Comment by kapi-no** [comment]
> > A documentation update is planned within the scope of this PR after the technical review (changelog + plugging the UUID API documentation into RST Fast Pair file).
> 
> Added changelog entries

Forgot to align the service module RST file and to add a new API to the relevant section. Now, this is addressed.

File: doc/nrf/libraries/bluetooth_services/services/fast_pair.rst [hunk 1]
```diff
@@ -132,9 +132,19 @@ The pairing flow and the set of Bluetooth authentication callbacks in use depend
...
-| Header file: :file:`include/bluetooth/services/fast_pair.h`
+| Header file: :file:`include/bluetooth/services/fast_pair/fast_pair.h`
 | Source files: :file:`subsys/bluetooth/services/fast_pair`
 
 .. doxygengroup:: bt_fast_pair
    :project: nrf
    :members:
+
+Fast Pair UUID API
+======================
```
Comment (divipillai) [inline doc/nrf/libraries/bluetooth_services/services/fast_pair.rst:143]:
```suggestion
Fast Pair UUID API
==================
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -792,12 +792,16 @@ Bluetooth libraries and services
 
 * :ref:`bt_fast_pair_readme` library:
 
-  * Added the :c:func:`bt_fast_pair_enable`, :c:func:`bt_fast_pair_disable`, :c:func:`bt_fast_pair_is_ready` functions to handle the Fast Pair subsystem readiness.
+  * Added:
+
+    * The :c:func:`bt_fast_pair_enable`, :c:func:`bt_fast_pair_disable`, :c:func:`bt_fast_pair_is_ready` functions to handle the Fast Pair subsystem readiness.
     In order to use the Fast Pair service, the Fast Pair subsystem must be enabled with the :c:func:`bt_fast_pair_enable` function.
+    * The :file:`ncs/nrf/include/bluetooth/services/fast_pair/uuid.h` header file that contains UUID definitions for the Fast Pair service and its characteristics.
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:799]:
```suggestion
  * Added:

    * The :c:func:`bt_fast_pair_enable`, :c:func:`bt_fast_pair_disable`, :c:func:`bt_fast_pair_is_ready` functions to handle the Fast Pair subsystem readiness.
      In order to use the Fast Pair service, the Fast Pair subsystem must be enabled with the :c:func:`bt_fast_pair_enable` function.
    * The :file:`include/bluetooth/services/fast_pair/uuid.h` header file that contains UUID definitions for the Fast Pair service and its characteristics.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -792,12 +792,16 @@ Bluetooth libraries and services
...
 
   * Updated:
 
     * Improved the :ref:`bt_fast_pair_readme` library documentation to include the description of the missing Kconfig options.
+    * The location of the :file:`ncs/nrf/include/bluetooth/services/fast_pair.h` header file and moved it to the :file:`ncs/nrf/include/bluetooth/services/fast_pair/` directory to have a common place for Fast Pair headers.
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:804]:
```suggestion
    * The :file:`include/bluetooth/services/fast_pair.h` header file is moved to the :file:`ncs/nrf/include/bluetooth/services/fast_pair` directory to have a common place for Fast Pair headers.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst (hunk 2 above)
Comment (divipillai) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:799]: This fixes the doc build error.

**Review by MarekPieta — APPROVED** [review]

**Review by divipillai — APPROVED** [review]

**Pull request merged**